specs/waku/waku-1.md

```diff
@@ -303,7 +302,15 @@ Each node SHOULD broadcast its rate limits to its peers using the rate limits pa
 
 Each node SHOULD respect rate limits advertised by its peers. The number of packets SHOULD be throttled in order not to exceed peer's rate limits. If the limit gets exceeded, the connection MAY be dropped by the peer.
 
-##### Message Confirmations Field
+##### light Node Field
+
+Light Node represents a boolean option. When true, the peer SHOULD NOT forward Envelopes through the Messages packet (`0x01`).
+
+##### Confirmations Enabled Field
+
+Confirmations enabled represents a boolean option. When true, the peer SHOULD send [message confirmations](#batch-acknowlege-and-message-response).
+
+#### Batch Acknowlege and Message Response
```

Comment:
Can we call it Batch Ack as that's how its called in the rest of the spec. Or rename all to Batch Acknowledge.

Reply:
Ok, maybe I can change the first instance of "Batch Ack" to "Batch Acknowledge (Batch Ack)" and then keep all other instances as "Batch Ack". WDYT?

Reply:
Well, while I'd normally be fine with that, the issue is perhaps that `batch-ack` in the ABNF spec is the first instance.

Reply:
That's a good point. What about adding Batch Ack to definitions?